modules/openapi-generator/src/main/resources/typescript-axios/package.mustache

```diff
@@ -17,8 +17,12 @@
   "license": "Unlicense",
   "main": "./dist/index.js",
   "typings": "./dist/index.d.ts",
+{{#supportsES6}}
+  "module": "./dist/esm/index.js",
+  "sideEffects": false,
+{{/supportsES6}}
   "scripts": {
-    "build": "tsc --outDir dist/",
+    "build": "tsc {{#supportsES6}}&& tsc -p tsconfig.esm.json{{/supportsES6}}",
```

Comment:
please keep the existing behavior by adding `{{^supportsES6}}--outDir dist/{{/supportsES6}}`
```suggestion
    "build": "tsc {{#supportsES6}}&& tsc -p tsconfig.esm.json{{/supportsES6}}{{^supportsES6}}--outDir dist/{{/supportsES6}}",
```

Reply:
isn't it already provided through https://github.com/OpenAPITools/openapi-generator/blob/94bc2ed19c75caf627a58f48d9cb022669292bb7/modules/openapi-generator/src/main/resources/typescript-axios/tsconfig.mustache#L7 though?

Reply:
Thanks Julien

Yes, this is why I removed the `outDir`. We don't need it in the `package.json` file, as it's already mentioned in the `tsconfig`.

I already tested generating TS clients on my local 👍

Reply:
makes sense, thx